### PR TITLE
Mailroom-Pt2, Exceptions-exercise for submission

### PR DESCRIPTION
**Pull request by nichx313**
submitting mailroom-pt2.py and exceptions_exercise.py for grading, no grades yet on already submitted - slicing_lab.py, strformat.py (also submitting it edX). Most other commits are sketchpad files or small updates not ready to grade.

### CODE REVIEW

File: students/nick_miller/lesson04/mailroom-pt2.py [hunk 1]
```diff
@@ -0,0 +1,242 @@
+#!/usr/bin/env python3
+
+"""PY210_SP - mailroom part 2
+author: Nick Miller"""
+
+import sys
```
Comment (charlesrobison) [inline students/nick_miller/lesson04/mailroom-pt2.py]: sys imported but not used

File: students/nick_miller/lesson04/mailroom-pt2.py [hunk 2]
```diff
@@ -0,0 +1,242 @@
...
+
+menu_dict = {"1": one_thanks,
+             "2": thanks_all,
+             "3": report,
+             "4": quit_prog,
+             "q": quit_prog,}
+
+
+if __name__ == "__main__":
+    menu_selection(main_prompt, menu_dict)
```
Comment (charlesrobison) [inline students/nick_miller/lesson04/mailroom-pt2.py]: Works great!

**Comment by charlesrobison** [comment]
> submitting mailroom-pt2.py and exceptions_exercise.py for grading, no grades yet on already submitted - slicing_lab.py, strformat.py (also submitting it edX). Most other commits are sketchpad files or small updates not ready to grade.

@nichx313 - which PR were slicing_lab.py, strformat.py on?  The submittal on the course site has 678, but I don't see the code.  Can you please confirm?

**Comment by natasha-aleksandrova** [comment]
@nichx313 please follow up with us via this PR (it will get merged) or via email. Thanks.

**Comment by nichx313** [comment]
Hmm. The code should be current now. I can do a new PR for those two if need be. Let me know.

**Comment by nichx313** [comment]
I’ve re-updated the strformat and slicing labs in this latest PR https://github.com/UWPCE-PythonCert-ClassRepos/SP_Online_PY210/pull/709 <https://github.com/UWPCE-PythonCert-ClassRepos/SP_Online_PY210/pull/709>, if you could link those, or delete my previous submission and I’ll turn them in again. Thanks!

**Pull request merged**